packages/components/package.json

```diff
@@ -22,6 +22,7 @@
 	"react-native": "src/index",
 	"dependencies": {
 		"@babel/runtime": "^7.4.4",
+		"@storybook/addon-knobs": "5.2.4",
```

Comment:
it's a development dependency, it shouldn't be included here, because all npm package consumers will have to download it for no reason.

Reply:
ok, moved it to dev, and fixes it.

I had it as dev dependency in the top package.json but the lint rule kept complaining until I added it to the components package.json.

Reply:
In here, we will have include `stories` folder as well:
https://github.com/WordPress/gutenberg/blob/d7cd263748b2c02b5509f86afcc42546c2fb6a41/.eslintrc.js#L126

Reply:
Something like this
```js
'**/@(storybook|stories)/**/*.js',
```

Then we don't need to define it as a dev dependency.

Reply:
:+1: